tests/integration/requirements.txt

```diff
@@ -1,2 +1,3 @@
 ansible==2.0.1
 dnspython==1.15.0
```

Comment:
I tried that with 21a33aa, but got [test failures|https://circleci.com/gh/open-craft/ansible-load-balancer/47?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link], so added it back in.